tests/mock_unit/files/udf_file.py

```diff
@@ -1,6 +1,9 @@
 #
 # Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
 #
+import pytest
+
+pytestmark = pytest.mark.skip
```

Comment:
Why skip?

Reply:
yes, if not adding this mark, this is picked up as a test file.
or it can be ignore using --ignore option in tox file, but in my test it leads to the file missing when we try to register this file for a udf.